Sources/Brave/Frontend/Sync/SyncWelcomeViewController.swift

```diff
@@ -228,15 +228,30 @@ class SyncWelcomeViewController: SyncViewController {
         self.navigationController?.pushViewController(view, animated: true)
       }
 
+      self.syncAPI.leaveSyncGroup()
```

Comment:
Why do we need this one?

Reply:
Wanted to be sure observers are removed and chain is reseted before creation but we dont need it, Removed at 
https://github.com/brave/brave-ios/pull/7156/commits/7d94bfb9082599c0840a79bfb6ec2eb30c7cbc09